src/main/java/edu/harvard/iq/dataverse/util/FileUtil.java

```diff
@@ -239,11 +239,12 @@ public static String getFacetFileType(DataFile dataFile) {
                 // we are going to try to add capitalization here to this function
                 // in order to capitalize all the unknown types that are not called
                 // out in MimeTypeFacets.properties
-                return fileType.split("/")[0];
+                String typeClass = fileType.split("/")[0];
+                return Character.toUpperCase(typeClass.charAt(0)) + typeClass.substring(1);
```

Comment:
I'm confused about why this capitalization is happening in this "MissingResourceException" catch. I assumed the plan would be to adjust the case in  MimeTypeFacets.properties to be whatever consistent naming we want. Right now there are 50 or so lowercase entries with no spaces like "networkdata" and only 1 with title case, which is "Tabular Data".

Reply:
Oh, I was only looking at 6a4c008 by @landreev and not a195d3b by @mheppler which has the change I was expecting. Sorry to botch this review. I do think we should remove the English word "Unknown" from the Java code, though, and move it to a bundle, so it can be translated.